_posts/this-week-in-solid/next.md

```diff
@@ -12,18 +12,29 @@ author:
 
 ### [Events](https://solidproject.org/events)
 
-In light of the COVID-19 pandemic Solid Events are moving online for the foreseeable future.
+Solid Events are open gatherings for anyone interested in Solid; run and organised by people like you. While these events used to happen in person in cities around the world, Solid Events are moving online for the foreseeable future in light of the COVID-19 pandemic. Solid World is an opportunity to meet people who are working on or interested in working on Solid.
 
-[Tune in](https://zoom.us/j/606342071) on the 2nd April 2020 at 1600 CEST to join.
+[Read more about Solid World April](https://www.eventbrite.com/e/solid-world-tickets-100417571660) which is happening this Thursday 2nd at 1600 CEST.
 
-If you had upcoming Solid Events planned reach out to the Solid Manager on info@solidproject.org to arrange moving the content online with Solid World.
+If you have any questions for the presenters [pop them in this thread](https://forum.solidproject.org/t/solid-world-april-2020/2788).
 
 Are you interested in speaking about what you have been building on Solid? Reach out to the Solid Manager on info@solidproject.org to get a speaking slot at the next event.
 
-There will be an online Solid Event every first Thursday of the month.
+### Covid-19 Hackathon
 
+Come take part in the MIT Covid-19 Hackathon on the 3rd April until 5th April. This is a great opportunity to share Solid expertise and values and apply them to an urgent and humanitarian use case. Solid was born out of an MIT project, so it’s fitting for us to contribute to this hackathon. Hope to see you there next weekend!
+
+## Solid Implementation
+
+If you have any new [apps](https://github.com/solid/solid-apps), [Pods](https://github.com/solid/pods), or [identity providers](https://github.com/solid/solid-idp-list), or if there are updates to existing Solid implementations, make sure to add them to the list so they get included in next week's edition.
+
+* [Golf Companion app](https://gca-solid.now.sh/login)
+
+### Talks & Articles
 Have you seen any articles or talks about Solid this week? If you've seen any talks, articles, or written blog posts about Solid please do send them over for next week.
 
+* [Co-Immunology and the Web](https://medium.com/@bblfish/co-immunology-and-the-web-43379b46688e) by [Henry Story](https://medium.com/@bblfish)
+
 ### Job Openings
-* [Digita is hiring](https://www.digita.ai/careers) a Full Stack Developer and a Linked Data Principal. Email tom@digita.ai for more information
-* [Inrupt is hiring](https://inrupt.com/careers) a Head of Business Development, DevOps Engineer, and Senior Front-End Engineer/Architect.)
+* [Digita is hiring](https://www.digita.ai/careers) a Full Stack Developer and a Linked Data Principal. Email tom@digita.ai for more information.
+* [Inrupt is hiring](https://inrupt.com/careers) a DevOps Engineer and Technical Writer. Email jobs@inrupt.com for more information.
```

Comment:
Make e-mail addresses hyperlinks?

Reply:
Huh, I thought Jekyll would auto-link them, but apparently not. I'll fix that after this one (since this file isn't public yet anyway).